### ISSUE TITLE
warnings on start

### ISSUE THREAD

**Issue by hvanaelst** [codeforboston/communityconnect#139]
react-scripts start gives the following warnings:

./src/components/ShoppingCart.js
  Line 33:  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji
  Line 40:  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji

./src/data/sheetLoadingHelpers.js
  Line 5:  'element["name"]' is assigned to itself  no-self-assign

./src/components/SavedResource/SavedResource.js
  Line 75:  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji
  Line 95:  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji

./src/components/OrganizationCard.js
  Line 91:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank

**Comment by hvanaelst** [comment]
Resolved in https://github.com/codeforboston/communityconnect/pull/140
